### ISSUE TITLE
Recommended best practice for maxAge minAge

### ISSUE THREAD

**Issue by dennereed** [VertNet/dwc-chronology#5]
What is your recommendation for using these terms in the case where a fossil specimen is not dated directly. Instead dates apply to the context. How then shall we apply minAge maxAge and chronometricAgeUncertainty? For example, a specimen derives from somewhere in the Lower Laetolil Beds at Laetoli, TZ. This deposit has K-Ar dates for the upper most and lower most marker tuffs of 3.627 +- 0.018 and 3.85 +- 0.02 respectively.  Is the maxAge 3.85 or 3.87 Ma?  If there's only a single term for uncertainty how shall it be applied when values differ? Best to use the largest uncertainty, e.g. 0.02 in this example?

**Comment by tucotuco** [comment]
@dennereed Do you feel there is a need for separate uncertainties around minimum and maximum ages? If they aren't really the same, it seems like maybe there is, to be able to be rigorous.

**Comment by dennereed** [comment]
@tucotuco Yes, adding separate uncertainties for min and max ages would help.

**Comment by lbrensk** [comment]
There was an example case we had from Edward Davis where the specimen was dated by being sandwiched between two dated tuff layers. In reporting this, I also had problems with reporting uncertainty and the material dated. I split the uncertaintyMethods and materialDated into these subfields, but perhaps we should consider splitting these terms for max & min ages? Is there a neat way to do this, @tucotuco , where it doesn't make things so ugly when only one method was used? That is my main concern; it may make things more confusing when you want to report a more traditionally dated specimen?

minimumChronometricAgeUncertaintyInYears: 0.016 Ma
maximumChronometricAgeUncertaintyInYears: 0.05 Ma

maximumAgeMaterialDated:Double Tuff
minimumAgeMaterialDated: Hawk Rim Tuff

**Comment by tucotuco** [comment]
I think this is the way to have all of the information in the more complex case. I would fill in all fields always, even when there is one MaterialData for both.

**Comment by emerykf** [comment]
I agree that the chronology extension would be well served by having uncertainty for both max and min dates. But verbatim and notes fields can also be used to provide more detailed information in any situation where it isn't clear what best practice should be. For e.g. deenereed your verbatim field in this case could say "Specimen derives from somewhere in the Lower Laetolil Beds at Laetoli, TZ. This deposit has K-Ar dates for the upper most and lower most marker tuffs of 3.627 +- 0.018 and 3.85 +- 0.02 respectively"
My hunch, had the specimen been mine, would be to put the max/min dates as given (3.627 and 3.85)  and then the uncertainty as the sum of the two ranges, but I think without having the verbatim information, it would be difficult then for a data-user to know what decision had been made for the input.

**Comment by neillwallisGitHub** [comment]
In terms of radiocarbon (and probably all absolute dating methods?), I’d expect most users will think of the minimum and maximum ages as the upper and lower limits of the 2-sigma range, not the 1-sigma “error” or “uncertainty.” With radiocarbon at least (and probably K-Ar?), the convention is to report with the standard error (1-sigma). But that is not conservative enough for min and max ages. For example, for a date reported as 1000 +/- 30 radiocarbon years before present we’d want to give radiocarbon ages of 940 BP (min) and 1060 BP (max), not 970 BP and 1030 BP.  I’d also wager that calibrated calendar years will be more useful for searching for than radiocarbon years, so it would be helpful to have that included as well.

**Comment by visead** [comment]
How would you propose handling complex ranges derived from calibrated radiocarbon dates? 
In the example below the 'easiest' range is AD 1575 to present, but the statistically 'probable' range is made up of 5 separate min-max spans.

![image](https://user-images.githubusercontent.com/35995002/78120850-e6450780-740a-11ea-8bf6-bcc42fe8b2eb.png)

**Comment by lbrensk** [comment]
@visead We are leaving it up to data providers to decide what range/statistical uncertainty they want to report, but providing a reference where these different ranges are reported would be advantageous so data users can decide for themselves what range they prefer for their purposes. Alternatively, you could report all these ranges with their uncertainties as different chronometricAge entries for the same specimen(s).
